traffic_ops/testing/api/v4/user_test.go

```diff
@@ -454,6 +454,12 @@ func GetTestUsers(t *testing.T) {
 	if err != nil {
 		t.Errorf("cannot get users: %v - alerts: %+v", err, resp.Alerts)
 	}
+	if resp.Response == nil {
+		t.Fatalf("expected a users list, got nothing")
+	}
+	if resp.Response[0].LastAuthenticated == nil {
```

Comment:
This line will still segfault if Traffic Ops returns an empty list - if you just check for `len(resp.Response) < 1` or similar it will also handle `nil` properly since `len([]string(nil)) == 0`

Reply:
Ugghh.. was thinking len() but then did nil..